srv/log2alsng.js

```diff
@@ -32,15 +32,24 @@ module.exports = class AuditLog2ALSNG extends AuditLogService {
     }[event]()
   }
 
+  flattenAndSortIdObject(id) {
+    if (!id || JSON.stringify(id) === '{}') return 'not provided'
+
+    return Object.entries(id)
+      .sort(([keyA], [keyB]) => keyA.localeCompare(keyB))
+      .map(([key, value]) => `${key}:${value}`)
+      .join(' ')
```

Comment:
```suggestion
    let s = ''
    for (const k of Object.keys(id).sort()) s += `${k}:${id[k]} `
    return s.trim()
```
more readable to me and should be faster